app/models/user.rb

```diff
@@ -13,6 +13,15 @@ class User < ApplicationRecord
   validate :birthday_within_100_years
   validates :gender, presence: true
 
+  def self.digest string
+    cost = if ActiveModel::SecurePassword.min_cost
+             BCrypt::Engine.MIN_COST
```

Comment:
```suggestion
             BCrypt::Engine::MIN_COST
```
Sai syntax khi gọi biến const

Reply:
<img width="525" height="205" alt="image" src="https://github.com/user-attachments/assets/b61a51bb-8a98-4c52-a9e4-958920bc5a71" />